api/api_full.go

```diff
@@ -863,6 +863,13 @@ type FullNode interface {
 	// Returns the client version
 	Web3ClientVersion(ctx context.Context) (string, error) //perm:read
 
+	// TraceAPI related methods
+	//
+	// Returns traces created at given block
+	TraceBlock(ctx context.Context, blkNum string) (interface{}, error) //perm:read
```

Comment:
Rename to `EthTraceBlock`?

Reply:
And is there a reason we're returning interfaces here? Can we not say we return `[]*TraceBlock`?

Reply:
> Rename to `EthTraceBlock`?
I considered doing that, but skipped the Eth prefix since the method is called "trace_block" and not "eth_traceBlock", but otherwise do not strongly prefer one or the other. Let me refactor the interface as per you latter comment and if it makes sense to rename I will do that

> And is there a reason we're returning interfaces here? Can we not say we return []*TraceBlock?
Definitely, good point!